<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvaluationSheetEditDialog</class>
 <widget class="QDialog" name="EvaluationSheetEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluation Sheet</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>1. Auswerter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DNA/RNA#:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_dna_rna"/>
      </item>
      <item row="3" column="2">
       <widget class="DBSelector" name="ds_reviewer2"/>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="DBSelector" name="ds_reviewer1">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>2. Auswerter:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Auswerteumfang:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Datum:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="de_review1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Datum:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="de_review2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ACMG:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cb_acmg_requested">
          <property name="text">
           <string>angefordert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_acmg_analyzed">
          <property name="text">
           <string>analysiert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_acmg_suspicious">
          <property name="text">
           <string>auffällig</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="cb_analysis_scope">
        <item>
         <property name="text">
          <string>voll (Single)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>voll (Trio + Single)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>voll (Multi + Single)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>beschränkt auf Zielregion</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Filterung erfolgt</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="cb_filter_freq_dom">
          <property name="text">
           <string>Freq.-basiert dominant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_rec">
          <property name="text">
           <string>Freq.-basiert rezessiv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_cnv">
          <property name="text">
           <string>CNV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_mito">
          <property name="text">
           <string>Mitochondrial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_x_chr">
          <property name="text">
           <string>X-chromosomal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_phen">
          <property name="text">
           <string>Phänotyp-basiert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_multi">
          <property name="text">
           <string>Multi-Sample-Auswertung</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_trio_stringent">
          <property name="text">
           <string>Trio stringent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_filter_trio_relaxed">
          <property name="text">
           <string>Trio relaxed</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>248</width>
       <height>148</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">DBSelector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_dna_rna</tabstop>
  <tabstop>ds_reviewer1</tabstop>
  <tabstop>de_review1</tabstop>
  <tabstop>ds_reviewer2</tabstop>
  <tabstop>de_review2</tabstop>
  <tabstop>cb_filter_freq_dom</tabstop>
  <tabstop>cb_filter_rec</tabstop>
  <tabstop>cb_filter_cnv</tabstop>
  <tabstop>cb_filter_mito</tabstop>
  <tabstop>cb_filter_x_chr</tabstop>
  <tabstop>cb_filter_phen</tabstop>
  <tabstop>cb_filter_multi</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EvaluationSheetEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
